package com.desafioitau.api.transferencia.v1.notificacao.exception;

public class NotificacaoTentativasExcedidasException extends NotificacaoException {

    public NotificacaoTentativasExcedidasException(Throwable cause) {
        super(cause);
    }
}
